### ISSUE TITLE
I think bug https://github.com/ToolJet/ToolJet/issues/5723 is back.

### ISSUE THREAD

**Issue by mgale** [ToolJet/ToolJet#10621]
### Environment
Running local docker env, using:
* `tooljet/tooljet-ce:CE-LTS-latest`
* Also tried ee version

### What is the expected behaviour?
Body attribute is available.

### What is the current behaviour?
No headers, params, or body attribute is avalable.

### How to reproduce the issue?
Add a endpoint to OpenAPI datasource, I tried using the default petstore api and a petstore openapi from https://github.com/go-fuego/fuego/tree/main/examples/petstore

### Screenshots or Screencast
First screenshot is the OpenAPI and 2nd screenshot is the RESTAPI
![Screenshot 2024-08-12 at 4 48 19 PM](https://github.com/user-attachments/assets/191816ad-1f8a-4142-9ab4-5994949e96d3)
![Screenshot 2024-08-12 at 4 48 35 PM](https://github.com/user-attachments/assets/94abbf35-1fd4-44bc-a8db-4a1f80f42bda)

**Comment by SRIKARREDDY-dotorg** [comment]
I will work on this issue

**Comment by PriteshKiri** [comment]
Hey @SRIKARREDDY-dotorg , this issue will be handled by our internal team. 

Please check out other open issues under the `community contribution` label.

**Comment by briankariuki** [comment]
Any update on this?

I cannot pass query parameters on OpenAPI datasource

**Comment by evelant** [comment]
It looks like this is still broken almost 2 years later. I'm evaluating ToolJet for our needs and it seems far too buggy to be worth the time, especially if basic things like this go broken and unaddressed for 2 years.

**Comment by evelant** [comment]
I'm pretty sure I know what the bug is -- ToolJet doesn't parse valid OpenAPI specs. It seems to only parse extremely basic ones. You can pass it a valid spec compliant definition and it will silently just fail to include endpoint parameters. If you "dumb it down" to a hello world simple spec it will pick up the parameters. It appears to be a very half baked feature.

**Comment by shubh22** [comment]
Hi @evelant ,
Yes, we have not scaled our OpenAPI integration as of now and has some issues for sure. But we are now improving it. 
If it is possible, could you share some example Specs you are trying to pass? If not exact, if you can share some redacted version also without actual values, just the structure would also work.
